src/YoutubeAtomPlayer.tsx

```diff
@@ -237,164 +238,151 @@ export const YoutubeAtomPlayer = ({
         hasSent75Event: false,
         hasSentEndEvent: false,
     });
-    const playerListeners = useRef<Array<YoutubeCallback>>([]);
-
-    /**
-     * A map ref with a key of eventname and a value of eventHandler
-     */
-    const customListeners = useRef<
-        Record<string, (event: CustomEventInit<CustomPlayEventDetail>) => void>
-    >({});
     const id = `youtube-video-${uniqueId}`;
 
     /**
-     * Initialise player useEffect
+     * Initialise player
+     *
+     * Please refer to the useOnMount API
+     *
+     * The following dependencies are used with the values set at component mount:
```

Comment:
This looks really good. Would you mind explaining how the dependancies are used while we're not adding them to any dependancy array?